app/views/languages/repository.erb

```diff
@@ -0,0 +1,67 @@
+<div class="container">
+  <section class="contributing-intro jumbotron">
+    <h1>Exercism Repositories</h1>
+
+    <p class="lead">
+      <strong>What are the repositories you can contribute to?</strong><br />
+    </p>
+
+    <p> Exercism is a project comprised of several repositories and these can be divided basically into:</p>
+
+     <h2> Basic components </h2>
```

Comment:
Why do we have both `app/views/languages/repository.erb` and `app/views/languages/repositories.erb`?